src/AssetRegistry/ATokenPricingModule.sol

```diff
@@ -26,26 +26,34 @@ contract ATokenPricingModule is PricingModule {
     struct AssetInformation {
```

Comment:
Inconsistent, sometimes we keep the address suffix sometimes we removed it

Reply:
within the struct? seems to be always there, we can change it to `address asset` though

Reply:
will do once in dev